code/modules/mob/living/basic/heretic/maid_in_the_mirror.dm

```diff
@@ -33,48 +33,16 @@
 	)
 	AddElement(/datum/element/death_drops, loot)
 	GRANT_ACTION(/datum/action/cooldown/spell/jaunt/mirror_walk)
+	ADD_TRAIT(src, TRAIT_UNHITTABLE_BY_LASERS, INNATE_TRAIT)
 
 /mob/living/basic/heretic_summon/maid_in_the_mirror/death(gibbed)
 	var/turf/death_turf = get_turf(src)
 	death_turf.TakeTemperature(-40) // Spooky
 	return ..()
 
-// Examining them will harm them, on a cooldown.
-/mob/living/basic/heretic_summon/maid_in_the_mirror/examine(mob/user)
```

Comment:
This makes me quite sad to see go, I'd rather the damage number be tweaked down (even to, like, 2%) than lose the flavor here.

Reply:
what if instead it slows them down or pushes them away a tile? might be interesting